src/middlewares/whitelist.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction } from 'express';
+import { getRequestInfo } from '../utils/users';
+import { isAuthenticated } from './authentication';
+import whitelist from '../../whitelist.json';
+import { forbidden } from '../utils/responses';
+import { Error } from '../types';
+
+export default [
+  ...isAuthenticated,
+  (request: Request, response: Response, next: NextFunction) => {
+    const { user } = getRequestInfo(response);
+    const { tournamentId } = request.body;
+    if (tournamentId in whitelist) {
```

Comment:
```suggestion

    if (tournamentId in whitelist) {
```
C'est bien d'aérer :)